.github/actions/pytest-with-rerun/action.yml

```diff
@@ -15,9 +15,12 @@ runs:
       with:
         command: test--${{ inputs.test-type }}
         requires-aws: ${{ inputs.requires-aws }}
+        restore-from-cache: "false"
+        restore-third-party-cache: "true"
 
     - uses: ./.github/actions/make/
       with:
         command: test--${{ inputs.test-type }}--rerun PYTEST_FLAGS="--suppress-no-test-exit-code"
         requires-aws: ${{ inputs.requires-aws }}
         restore-from-cache: "false"
+        restore-third-party-cache: "true"
```

Comment:
don't think we want this here

Reply:
If you take it away it cant run pytest or behave as it fails to locate the package - likely because the .venv is part of the cache